From 3965306ee734251595485365f0f3cff6a22dc0f2 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Mon, 25 Jan 2016 13:59:25 +0900
Subject: [PATCH 1127/2848] HADOOP-12736.
 TestTimedOutTestsListener#testThreadDumpAndDeadlocks
 sometimes times out. Contributed by Xiao Chen.

(cherry picked from commit 643227927a7d7974655627d7e97aae42600692ae)
(cherry picked from commit 38ec290455a61a53f1f2c8fcc06f914de52fd5e2)

Change-Id: Idd3136600b744b4ae90ebf5d8283004d5ddc4b55
---
 .../hadoop/test/TestTimedOutTestsListener.java     |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/test/TestTimedOutTestsListener.java b/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/test/TestTimedOutTestsListener.java
index 62748b4..1334f1c 100644
--- a/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/test/TestTimedOutTestsListener.java
+++ b/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/test/TestTimedOutTestsListener.java
@@ -144,7 +144,7 @@ private void goMonitorDeadlock() {
   
   }
 
-  @Test(timeout=500)
+  @Test(timeout=30000)
   public void testThreadDumpAndDeadlocks() throws Exception {
     new Deadlock();
     String s = null;
-- 
1.7.9.5

